Dockerfile

```diff
@@ -21,4 +21,5 @@ RUN wget https://download.asperasoft.com/download/sw/connect/3.8.1/ibm-aspera-co
 
 RUN cd /root; git clone https://github.com/enasequence/enaBrowserTools.git; mv enaBrowserTools /enaBrowserTools
 
-ENV PATH /enaBrowserTools/python3:/opt/conda/envs/nf-core-bactmap-1.0dev/bin:$PATH
\ No newline at end of file
+ENV PATH /opt/conda/envs/nf-core-bactmap-1.0dev/bin:$PATH
+ENV PATH /enaBrowserTools/python3:$PATH
```

Comment:
```suggestion
```

Reply:
There is a bioconda package for this: see https://bioconda.github.io/recipes/enabrowsertools/README.html

Reply:
Thanks @apeltzer, good call. I've pushed this update